docs/docs/about/migration_guides/v8.26_to_v9.0.md

```diff
@@ -0,0 +1,84 @@
+
+We try to make this as easy as possible, and provide a step-by-step migration guide.
+
+### 1. Changes to compressor trains in MODELS
+- `CONTROL_MARGIN` and `CONTROL_MARGIN_UNIT` are now required for `SINGLE_SPEED_COMPRESSOR_TRAIN`, `VARIABLE_SPEED_COMPRESSOR_TRAIN` and `VARIABLE_SPEED_COMPRESSOR_TRAIN_MULTIPLE_STREAMS_AND_PRESSURES`.
+
+An example with new yaml implementation is shown below:
+
+```yaml 
+MODELS:
+  - NAME: <model name>
+    TYPE: SINGLE_SPEED_COMPRESSOR_TRAIN
+    FLUID_MODEL: <reference to fluid model>
+    PRESSURE_CONTROL: <method for pressure control, DOWNSTREAM_CHOKE (default), UPSTREAM_CHOKE, , INDIVIDUAL_ASV_PRESSURE, INDIVIDUAL_ASV_RATE or COMMON_ASV>
+    MAXIMUM_DISCHARGE_PRESSURE: <Maximum discharge pressure in bar (can only use if pressure control is DOWNSTREAM_CHOKE)>
+    COMPRESSOR_TRAIN:
+      STAGES:
+        - INLET_TEMPERATURE: <inlet temperature in Celsius for stage>
+          COMPRESSOR_CHART: <reference to compressor chart model for first stage, must be defined in MODELS or FACILITY_INPUTS>
+          PRESSURE_DROP_AHEAD_OF_STAGE: <Pressure drop before compression stage [in bar]>
+          # This is new
+          CONTROL_MARGIN: <Default value is zero>
+          # This is new
+          CONTROL_MARGIN_UNIT: <FRACTION or PERCENTAGE, default is PERCENTAGE>
+        - INLET_TEMPERATURE: <inlet temperature in Celsius for stage>
+          COMPRESSOR_CHART: <reference to compressor chart model for second stage, must be defined in MODELS or FACILITY_INPUTS>
+          PRESSURE_DROP_AHEAD_OF_STAGE: <Pressure drop before compression stage [in bar]>
+          # This is new
+          CONTROL_MARGIN: <Default value is zero>
+          # This is new
+          CONTROL_MARGIN_UNIT: <FRACTION or PERCENTAGE, default is PERCENTAGE>
+        - ... and so forth for each stage in the train
+    POWER_ADJUSTMENT_CONSTANT: <Optional constant MW adjustment added to the model>
+    MAXIMUM_POWER: <Optional constant MW maximum power the compressor train can require>
+    CALCULATE_MAX_RATE: <Optional compressor train max standard rate [Sm3/day] in result if set to true. Default false. Use with caution. This will increase runtime significantly. >
+```
+
+### 2. Changes to simplified compressor trains in MODELS
+- `CONTROL_MARGIN` and `CONTROL_MARGIN_UNIT` are not allowed for `SIMPLIFIED_VARIABLE_SPEED_COMPRESSOR_TRAIN`.
+
+The new yaml implementation is described below, for simplified compressor train model with known compressor stages:
+
+```yaml 
+MODELS:
+  - NAME: <model name>
+    TYPE: SIMPLIFIED_VARIABLE_SPEED_COMPRESSOR_TRAIN
+    FLUID_MODEL: <reference to fluid model>
+    COMPRESSOR_TRAIN:
+      STAGES:
+        - INLET_TEMPERATURE: <inlet temperature in Celsius for stage>
+          COMPRESSOR_CHART: <reference to compressor chart model for first stage, must be defined in MODELS or FACILITY_INPUTS>
+          # This is old
+          CONTROL_MARGIN: <Default value is zero>
```

Comment:
This one should be removed, right?

Reply:
I think this is to show that they should be removed by the user in their models. The `# This is old` comment above control margin marks that line as red.

Reply:
@olelod : It is, as mentioned, to show what should be removed. It will be displayed like this:

![image](https://github.com/user-attachments/assets/f78837a4-f148-4cbe-8190-faf2adbe4eb0)

Reply:
I could of course specify that red lines show what should be removed, but not sure if we have done that other places?

Reply:
💡